front/lib/resources/vault_resource.ts

```diff
@@ -392,6 +392,10 @@ export class VaultResource extends BaseResource<VaultModel> {
     });
   }
 
+  /**
+   * Maps permissions to all groups associated with this vault, creating permission assignments for
+   * each group-vault relationship.
```

Comment:
What is permissions here? What's the purpose? Actually i don't see instantly where it's used?